src/store/modules/swap/staticRelays.ts

```diff
@@ -429,3 +429,20 @@ export const findPreviousPoolFee = (
   const res = previousFee && previousFee.oldDecFee;
   return res;
 };
+
+export const highTierPools = [
+  // mainnet
```

Comment:
Do you reckon ```highTierPools``` should be moved to type ```EthNetworkVariables``` getting returned with the func ```getNetworkVariables``` ? 

Makes the distinction between chains main net and ropsten more obvious + puts more of the hardcoded parts of the app in one place.

Reply:
yes i thought about this, then i decided to not take the effort because this should be supported by the contracts in the next versions very soon. So this is only temporary.